File: Codes/3222-find-the-winning-player-in-coin-game.go
/*
 * @Author: LetMeFly
 * @Date: 2024-11-05 09:55:16
 * @LastEditors: LetMeFly.xyz
 * @LastEditTime: 2024-11-05 09:56:06
 */
package main

func losingPlayer(x int, y int) string {
    if min(x, y / 4) % 2 == 1 {
        return "Alice"
    }
    return "Bob"
}

File: Codes/3222-find-the-winning-player-in-coin-game.py
'''
Author: LetMeFly
Date: 2024-11-05 09:52:45
LastEditors: LetMeFly.xyz
LastEditTime: 2024-11-05 09:53:17
'''
class Solution:
    def losingPlayer(self, x: int, y: int) -> str:
        return "Alice" if min(x, y // 4) % 2 else "Bob"